dev/buildtool/spin_commands.py

```diff
@@ -101,10 +107,21 @@ def build_all_distributions(self, repository):
       env.update({'CGO_ENABLED': '0',
                   'GOOS': dist_arch.dist,
                   'GOARCH': dist_arch.arch})
+
+      # Note: spin CLI is coupled to the Gate major and minor version.
+      # Gate is a routing server, so features and breaking changes in Gate
+      # must be reflected in spin since it is a client.
+      dash = self.__gate_version.find('-')
+      gate_semver = self.__gate_version[:dash]
+      # Unset ReleasePhase tag for proper versions.
+      prefix = 'github.com/spinnaker/spin/version'
```

Comment:
can you use repository.origin instead of the github.com/spinnaker/spin? That way you arent assuming the repo this is building.

Reply:
Yes, but it needs formatted as a go package for `ldflags`. Done.